Portal/src/Datahub.Portal/i18n/localization.fr.json

```diff
@@ -227,10 +227,10 @@
   "ArcGIS": "ArcGIS",
   "ArcGIS is a geographic information system (GIS) for working with maps and geographic information. It is used for creating and using maps, compiling geographic data, analyzing mapped information, sharing and discovering geographic information, using maps and geographic information in a range of applications, and managing geographic information in a database.": "ArcGIS est un système d'information géographique (SIG) pour travailler avec des cartes et des informations géographiques. Il est utilisé pour créer et utiliser des cartes, compiler des données géographiques, analyser des informations cartographiées, découvrir des informations géographiques dans une gamme d'applications.",
   "Architecture": "Architecture",
-  "Are you sure you want to delete file \\": "Etes-vous sûr de vouloir supprimer le fichier \\",
-  "Are you sure you want to delete folder \\": "Etes-vous sûr de vouloir supprimer le dossier \\",
+  "Are you sure you want to delete file \"{0}\"?": "Êtes-vous sûr de vouloir supprimer le fichier \"{0}\"?",
```

Comment:
technically there should be a space before the question mark

Reply:
In Canadian French there is no space, DeepL likes to play this trick on us.

[Article from the Office québécois de la langue française](https://vitrinelinguistique.oqlf.gouv.qc.ca/22039/la-typographie/espacement/espacement-avant-et-apres-les-signes-de-ponctuation-et-les-symboles)